chromatic/rainbows/readers/eureka_specdata.py

```diff
@@ -59,11 +59,11 @@ def from_eureka_SpecData(rainbow, filepath, optimal=True):
 
     # populate a 1D array of times (with astropy units of time)
     k = "time"
-    if dataset[k].attrs["time_units"] == "BJD_TDB":
+    if dataset[k].attrs["time_units"] == "BMJD_TDB":
```

Comment:
Thanks! I'm adding back in the BJD_TDB option if the BMJD_TDB can't be found, just in case folks have older eureka formats floating around. I added a warning about how it's being interpreted, so hopefully that will head off any confusion.